test/test.el

```diff
@@ -80,7 +80,7 @@
         (unless (> (point) eod)
           (goto-char (match-beginning 0))
           (forward-sexp)
-          (eval-last-sexp nil))))))
+          (ignore-errors (eval-last-sexp nil))))))) ;; to allow (should-error ...)
```

Comment:
Let's not make this change here.  If it's really important to allow `should-error` tests, we should do that separately.

Reply:
Yes, in hindsight it was a bit overenthusiastic. Now removed.